src/boot/efi/linux_x86.c

```diff
@@ -106,7 +106,6 @@ static void linux_efi_handover(EFI_HANDLE image, uintptr_t kernel, BootParams *p
         kernel += (params->hdr.setup_sects + 1) * KERNEL_SECTOR_SIZE; /* 32bit entry address. */
 
 #ifdef __x86_64__
-        asm volatile("cli");
```

Comment:
Hmmm. I have no idea how to evaluate this. I guess if things don't break, it must be ok ;)